typings/react-dropzone.d.ts

```diff
@@ -8,6 +8,11 @@ export interface DropzoneProps extends DropzoneOptions {
   children?(state: DropzoneState): JSX.Element;
 }
 
+export interface FileError extends Error {
+  message: string;
```

Comment:
Doesn't `Error` already have a message? But I would remove the extend as it's misleading and users might try to check w/ `instanceof` for errors.